marc.go

```diff
@@ -94,6 +100,8 @@ func marcToRecord(marcRecord *marc21.Record, rules []*Rule, languageCodes map[st
 	title := getFields(marcRecord, rules, "title")
 	if title != nil {
 		r.Title = title[0]
+	} else {
+		err = fmt.Errorf("Record %s has no title, check validity", r.Identifier)
```

Comment:
I wonder if we should just return the error at this point and stop processing the function. This is not a blocking comment as I think the code works fine as is.

Reply:
Oh, yeah we should definitely do that. Will update, thanks!